v2/internal/internalyarpcobservability/call.go

```diff
@@ -115,15 +115,7 @@ func (c call) endStats(elapsed time.Duration, err error, isApplicationError bool
 		return
 	}
 
-	if !yarpcerror.IsStatus(err) {
-		c.edge.serverErrLatencies.Observe(elapsed)
-		if counter, err := c.edge.serverFailures.Get(_error, "unknown_internal_yarpc"); err == nil {
```

Comment:
Do we have a changelog for expected behaviour changes yarpcv1 to v2? Would be good to attach that `unknown_internal_yarpc` metrics now change to `unknown`